pkg/configurator/client.go

```diff
@@ -55,6 +55,9 @@ const (
 
 	// enablePrivilegedInitContainer is the key name used to specify whether init containers should be privileged in the ConfigMap
 	enablePrivilegedInitContainer = "enable_privileged_init_container"
+
+	// configResyncInterval is a flag to configure resync interval for regular proxy broadcast updates
```

Comment:
For consistency with existing comments,
```suggestion
	// configResyncInterval is the key name used to configure the resync interval for regular proxy broadcast updates
```

Reply:
done